test/layers/recurrent_tests.jl

```diff
@@ -375,3 +375,85 @@ end
         @test Array(vec(reduce(vcat, res))) ≈ [0.5, 0.5, 1.2, 2.0]
     end
 end
+
+@testitem "BidirectionalRNN" timeout=3000 setup=[SharedTestSetup] tags=[:recurrent_layers] begin
+    rng = get_stable_rng(12345)
+    @testset "$mode" for (mode, aType, device, ongpu) in MODES
+        @testset "cell: $_cell" for _cell in (RNNCell, LSTMCell, GRUCell)
+            cell = _cell(3 => 5)
+            bi_rnn = BidirectionalRNN(cell)
+            bi_rnn_no_merge = BidirectionalRNN(cell; merge_mode=nothing)
+            __display(bi_rnn)
+
+            # Batched Time Series
+            x = randn(rng, Float32, 3, 4, 2) |> aType
+            ps, st = Lux.setup(rng, bi_rnn) .|> device
+            y, st_ = bi_rnn(x, ps, st)
+            y_, st__ = bi_rnn_no_merge(x, ps, st)
+
+            @jet bi_rnn(x, ps, st)
+            @jet bi_rnn_no_merge(x, ps, st)
```

Comment:
@avik-pal Hi, I have updated the implementation, but the `@jet` test fails and I can't find the reason. Can you please take a look at it when you have time?

Reply:
Yeah the tests wont really tell much here. Install `JET.jl` and run `@report_call bi_rnn_no_merge(x, ps, st) target_modules=[Lux, LuxLib, LuxCore]` and `@report_opt bi_rnn_no_merge(x, ps, st) target_modules=[Lux, LuxLib, LuxCore]`. This will give you a list of function calls that cause the test to fail

Reply:
@avik-pal Hi, I tried `@report_call bi_rnn_no_merge(x, ps, st) target_modules=[Lux, LuxLib, LuxCore]` but it report `@report_call expects only one non-keyword argument`, so I remove `target_modules=[Lux, LuxLib, LuxCore]`. And I replace `@jet` to `@report_call` and `@report_opt`, all test passed and no error throw.

Is this `LuxTestUtils` bug?,,, Could you please check this branch locally? Thank you very much!!

Reply:
no don't replace `@jet` with `report_*` functions, you need to run those 2 in the REPL / VSCode. That will point to the correct line. `@report_call target_modules=[Lux, LuxLib, LuxCore] bi_rnn_no_merge(x, ps, st)` https://aviatesk.github.io/JET.jl/dev/tutorial/#Analyse-methods-with-@report_call